components/create-element-modal/creation-section.tsx

```diff
@@ -11,24 +11,50 @@ interface AvailableElementsProps {
   onSelect: (elementType: string) => void;
 }
 
+interface HelpButtonProps {
+  documentationLink: string
+  description: string
+}
+
+function HelpButton(props: HelpButtonProps) {
+  const [hover, setHover] = useState(false)
+
+  const onMouseEnter = () => setHover(true)
+  const onMouseLeave = () => setHover(false)
+
+  console.log(hover)
+
+  // pq o hover nao ta funcionando? @davigsousa
+  return (
+    <div
+      onMouseEnter={onMouseEnter}
+      onMouseLeave={onMouseLeave}
+    >
+      {hover ?? <p>{props.description}</p>}
```

Comment:
Porque tem que ser && ao invés de ?? para retornar o lado direito, como o ?? é um OU, ele só vai fazer evaluate da primeira parte e nem vai rodar a segunda se for true